src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php

```diff
@@ -21,6 +21,26 @@
  */
 class TextDescriptor extends Descriptor
 {
+    protected function describeDefaults(array $options = array())
+    {
+        $coreTypes = $this->getCoreTypes();
+
+        $this->output->section('Built-in form types (Symfony\Component\Form\Extension\Core\Type)');
```

Comment:
not bound to `OutputInterface` but `StyleInterface`. Are we're being pragmatic here?

Reply:
Yep, we are subject to `DescriptorInterface` signature.

Reply:
Fixed.